geocity/apps/forms/models.py

```diff
@@ -762,6 +783,12 @@ class Field(models.Model):
         limit_choices_to={"permit_department__is_integrator_admin": True},
     )
     name = models.CharField(_("nom"), max_length=255)
+    api_name = models.CharField(
+        _("Nom dans l'API"),
+        max_length=255,
+        blank=True,
+        help_text=_("Se génère automatiquement lorsque celui-ci est vide."),
+    )
```

Comment:
Should be unique within a form in order no avoid duplicate keys in json output, no ?